goal_src/engine/target/logic-target.gc

```diff
@@ -2656,13 +2656,13 @@
    )
   (quaternion-identity! (-> self control unknown-quaternion01))
   (set! (-> self control unknown-float00) 0.0)
-  (set! (-> self control unknown-dword10) (-> *display* base-frame-counter))
+  (set! (-> self control unknown-dword11) (-> *display* base-frame-counter))
   (set! (-> self control unknown-float80) 0.0)
   (set! (-> self control unknown-float82) 32.0)
   (set! (-> self cam-user-mode) 'normal)
   (set! (-> self control unknown-handle10) (the-as handle #f))
   (set! (-> self control unknown-handle00) (the-as handle #f))
-  (set! (-> self control unknown-dword72) 0)
+  (set! (-> self control unknown-dword82) 0)
```

Comment:
we should be careful about renaming these if there are some files that use them that don't have offline tests.

Reply:
that's a good point ill keep that in mind (hopefully this type is mostly filled out now).  I checked the ones that got renamed and they are not outside the files that are covered / modified in this PR (mostly target related)

Reply:
yeah I also checked and I think you are good - just wanted to make sure we don't forget later on.